<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoDiagnose</class>
 <widget class="QDialog" name="AutoDiagnose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>UDS 协议</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>OTA刷写配置</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="startCANIDLabel">
              <property name="text">
               <string>启动CANID：0x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="startCANIDLineEdit_UDS">
              <property name="text">
               <string>18DAEEFA</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>数据：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="startDataLineEdit_UDS">
              <property name="minimumSize">
               <size>
                <width>160</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>02 FE 01 00 00 00 00 00</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>请求CANID：0x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="reqCANIDLineEdit_UDS">
              <property name="text">
               <string>18DA7FFA</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="rspCANIDLabel">
              <property name="text">
               <string>响应CANID：0x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="rspCANIDLineEdit_UDS">
              <property name="text">
               <string>18DAFA7F</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item>
             <widget class="QCheckBox" name="streamControlCheckBox">
              <property name="maximumSize">
               <size>
                <width>156</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>流控帧响应 - 帧间隔(ms)</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="streamIntervalLineEdit">
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>10</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>CCP 协议</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>CCP标定配置</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="startCANIDLabel_2">
              <property name="text">
               <string>启动CANID：0x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="startCANIDLineEdit_CCP">
              <property name="text">
               <string>0000010A</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>数据：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="startDataLineEdit_CCP">
              <property name="minimumSize">
               <size>
                <width>160</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>02 FE 01 00 00 00 00 00</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>请求CANID：0x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="reqCANIDLineEdit_CCP">
              <property name="text">
               <string>0100</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="rspCANIDLabel_2">
              <property name="text">
               <string>响应CANID：0x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="rspCANIDLineEdit_CCP">
              <property name="text">
               <string>0101</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>资源可用性：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="CALUseCheckBox">
              <property name="text">
               <string>CAL</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="PGMUseCheckBox">
              <property name="text">
               <string>PGM</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="DAQUseCheckBox">
              <property name="text">
               <string>DAQ</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>资源保护性：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="CALSaveCheckBox">
              <property name="text">
               <string>CAL</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="PGMSaveCheckBox">
              <property name="text">
               <string>PGM</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="DAQSaveCheckBox">
              <property name="text">
               <string>DAQ</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>XCP 协议</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>XCP标定配置</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="startCANIDLabel_4">
              <property name="text">
               <string>启动CANID：0x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="startCANIDLineEdit_XCP">
              <property name="text">
               <string>18DAEEFA</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>数据：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="startDataLineEdit_XCP">
              <property name="minimumSize">
               <size>
                <width>160</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>02 FE 01 00 00 00 00 00</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>请求CANID：0x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="reqCANIDLineEdit_XCP">
              <property name="text">
               <string>0100</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="rspCANIDLabel_3">
              <property name="text">
               <string>响应CANID：0x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="rspCANIDLineEdit_XCP">
              <property name="text">
               <string>0101</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
